### DISCUSSION TITLE
Holloween Loot Bags?

### DISCUSSION THREAD

**Discussion by Weru** [Taschenbuch/BlishHud-SessionTracker#5] [Ideas]
Would it be possible to track festival rewards - i.e. Holloween Loot Bags?

**Comment by Taschenbuch** [comment]
Update: trick-or-treat bags and piece of candy corn stats are available now after restarting the module. no module update required. No need to modify the model.json for it. But i will keep this guide here, for people who want to add other item stats.

update 2: the file format will change a bit in session tracker 9.0.0. So the example here is a bit outdated. You can find the new format by looking at session-tracker\data\format_version_4\model.json

yeah, session tracker can track any item already. They are currently just missing in the config file, because the current UI would be overwhelmed, when I add 600 material storage items for example. But if you want, you can add it yourself. 

# add trick-and-treat bag stat

1. Download the modified model.json: [model.json](https://github.com/Taschenbuch/BlishHud-SessionTracker/files/13060291/model.json)
1. turn off blish
1. go to Documents\Guild Wars 2\addons\blishhud\session-tracker\data\format_version_1
1. delete the model.json there and instead put the modified model.json there 
1. start blish again

- I did not test it, but it should work. 
- Warnings
  - there is another model.json directly in \blishhud\session-tracker. Ignore that one! only replace the one in \blishhud\session-tracker\data\format_version_1!
  - Just be aware, that a future module version may override the model.json file. 

# add other items as stat
If you want to add other items yourself, you can modify the model.json on your own like this. e.g. that is what i added for the trick and treat bag:

```json
{
  "Id": "item36038",
  "Name": {
    "LocalizedTextByLocale": {
      "English": "Trick-or-Treat Bag",
      "French": "",
      "German": "",
      "Spanish": "",
      "Chinese": "",
      "Korean": ""
    }
  },
  "Description": {
    "LocalizedTextByLocale": {
      "English": "",
      "French": "",
      "German": "",
      "Spanish": "",
      "Chinese": "",
      "Korean": ""
    }
  },
  "ApiId": 36038,
  "ApiIdType": "Item",
  "IconAssetId": 499375,
  "Value": {}
},
```

- "id" has to be unique, so give it another name if you add another item. like "abcd123".
- "ApiId" and "IconAssetId" can be found on the wiki page as described in the screenshot below.

![image](https://github.com/Taschenbuch/BlishHud-SessionTracker/assets/43114787/c5086069-1103-48bf-823a-0c8d48f0f797)

**Comment by Taschenbuch** [comment]
update: session-tracker 9.0.0 or later will include those by default.

model.json with fine, masterwork, rare unidentified gear:
[model.json](https://github.com/Taschenbuch/BlishHud-SessionTracker/files/14472725/model.json)
